lib/api/unittest/testfiles/instrumentation/outlier_detection_stats.schema.json

```diff
@@ -21,13 +21,15 @@
           "description": "Defines the value for how many nearest neighbors each method of outlier detection will use to calculate its outlier score.",
           "type": "integer"
         },
-        "methods": {
-          "description": "List of methods that outlier detection uses.",
-          "type": "array",
-          "items": [
-            {
-              "type": "string"
-            }
+        "method": {
+          "description": "Method that outlier detection uses.",
+          "type": "string",
+          "enum": [
+            "Lof",
+            "Ldof",
+            "DistancekNN",
+            "TotalDistancekNN",
+            "Ensemble"
```

Comment:
What about `Ensemble`, can I use `ensemble` here?

Reply:
I wonder if we should just omit if the type if ensemble. Since this is the default so not accepted as input. I think it is fine however to have print work for this, the special case handling could be in the instrumentation writing.